29.md

```diff
@@ -22,6 +22,12 @@ Relays are supposed to generate the events that describe group metadata and grou
 
 A group may be identified by a string in the format `<host>'<group-id>`. For example, a group with _id_ `abcdef` hosted at the relay `wss://groups.nostr.com` would be identified by the string `groups.nostr.com'abcdef`.
 
+Group identifiers must be strings restricted to the characters `a-z0-9-_`.
+
+When encountering just the `<host>` without the `'<group-id>`, clients can choose to connect to the group with id `_`, which is a special top-level group dedicated to relay-local discussions.
+
```

Comment:
here we can add UUID recommendations and why relays may reject some _id_.

Also, above we defined the _id_'s. here we defined a way to find a group using _id_ and _host_, combined. which I believe MUST be replaced with names in future.

Reply:
Then we can close #1493.

Reply:
UUID is excessively big and unnecessary, I don't think we should recommend it although some people may want to use it.

Reply:
We do not plan to use _id_'s for users at all. also on the other hand for example 0xchat is generating very big _id_'s larger than UUIDs.

Reply:
I think that is a bug in 0xchat then, and also relays should block giant ids.